httpservice/httpservice.go

```diff
@@ -0,0 +1,45 @@
+package httpservice
+
+import (
+	pwc "grab/week5/GrabGoTrainingWeek5Assignment/postwithcomment"
+	"log"
+	"net/http"
+)
+
+type RendererInterface interface {
```

Comment:
It's better to name interface ending by `er`  ( most case )
`type Render interface{ Render(value interface{}) error }`